guide/misc/osgi.md

```diff
@@ -0,0 +1,63 @@
+---
+title: FAQ
+layout: website-normal
+---
+
+# Running Apache Brooklin inside Karaf container
+
+The Apache Brooklyn Karaf based distribution lives in brooklyn-server/karaf/apache-brooklyn folder.
```

Comment:
Worth adding a note on the status of the Karaf distro, with caveats about not everything necessarily working yet.